docs/tutorials/05-model-monitoring.ipynb

```diff
@@ -2,452 +2,1478 @@
-    "training_set_path = mlrun.get_sample_path(\"data/model-monitoring/iris_dataset.csv\")"
+    "project.enable_model_monitoring(base_period=1)"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "6f1de530",
    "metadata": {},
    "source": [
-    "<a id=\"log-model\"></a>\n",
-    "## Log the model with training data\n",
+    "## Log the model artifacts\n",
+    "\n",
+    "See full parameter details in {py:meth}`~mlrun.projects.MlrunProject.log_model`.\n",
     "\n",
-    "Log the model using MLRun experiment tracking. This is usually done in a training pipeline, but you can also bring in your pre-trained models from other sources. See [Working with data and model artifacts](https://docs.mlrun.org/en/latest/training/working-with-data-and-model-artifacts.html) and [Automated experiment tracking](https://docs.mlrun.org/en/latest/concepts/auto-logging-mlops.html) for more information."
+    "First download {download}`the pickle file <./_static/model.pkl>`."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 3,
+   "id": "ee81d857-7c02-4687-bb3d-92f0f3e05c51",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<mlrun.artifacts.model.ModelArtifact at 0x7f2f1f7cadf0>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "model_name = \"RandomForestClassifier\""
+    "iris = load_iris()\n",
+    "train_set = pd.DataFrame(\n",
+    "    iris[\"data\"],\n",
+    "    columns=[\"sepal_length_cm\", \"sepal_width_cm\", \"petal_length_cm\", \"petal_width_cm\"],\n",
+    ")\n",
+    "\n",
+    "model_name = \"RandomForestClassifier\"\n",
+    "project.log_model(\n",
+    "    model_name,\n",
+    "    model_file=\"model.pkl\",\n",
+    "    training_set=train_set,\n",
+    "    framework=\"sklearn\",\n",
+    ")"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 5,
-   "metadata": {},
-   "outputs": [],
+   "cell_type": "markdown",
+   "id": "1f1993e5-94c8-472d-b026-e1f6e5f89297",
+   "metadata": {
+    "tags": []
+   },
    "source": [
-    "model_artifact = project.log_model(\n",
-    "    key=model_name,\n",
-    "    model_file=model_path,\n",
-    "    framework=\"sklearn\",\n",
-    "    training_set=pd.read_csv(training_set_path),\n",
-    "    label_column=\"label\",\n",
-    ")"
+    "## Import, enable monitoring, and deploy the serving function\n",
+    "\n",
+    "Import the [model server function](https://www.mlrun.org/hub/functions/master/v2-model-server/latest/example/)  from the MLRun Function Hub, add the model that was logged via experiment tracking, and enable drift detection.\n",
+    "\n",
+    "The core line here is `serving_fn.set_tracking()`, which creates the required infrastructure behind the scenes to perform drift detection. \n",
```

Comment:
it's not accurate. the `enable_model_monitoring()` API creates the infra behind the scenes. Applying `set_tracking()` on the serving function is indeed required but the meaning of this operation is that the relevant models will be tracked, and eventually we will be able to perform drift detection on these models.

Reply:
updated